### PR TITLE
add support for project tags

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: internal/service/project/resource_project.go [hunk 1]
```diff
@@ -68,6 +68,25 @@ var aivenProjectSchema = map[string]*schema.Schema{
 		Description:      schemautil.Complex("The id of the billing group that is linked to this project.").Referenced().Build(),
 		DiffSuppressFunc: schemautil.EmptyObjectDiffSuppressFunc,
 	},
+	"tag": {
```
Comment (dmitrii-vasilev) [inline internal/service/project/resource_project.go:71]: Why is it `tag` and not `tags`? It looks confusing below with `d.Set("tag", tags)`

File: internal/service/project/resource_project.go [hunk 2]
```diff
@@ -372,5 +393,27 @@ func setProjectTerraformProperties(d *schema.ResourceData, client *aiven.Client,
...
+	for k, v := range project.Tags {
+		tags = append(tags, map[string]interface{}{
+			"key":   k,
+			"value": v,
+		})
+	}
+	if err := d.Set("tag", tags); err != nil {
+		return diag.FromErr(err)
+	}
+
```
Comment (dmitrii-vasilev) [inline internal/service/project/resource_project.go:406]: Maybe this code should be extracted to a function? Then all properties in this function will have kind of the same set pattern. Something like the following:
```go
if err := setTags(d, project.Tags); err != nil {
    return diag.FromErr(err)
}
```

**Review by dmitrii-vasilev — CHANGES_REQUESTED** [review]
A couple of comments / questions

File: internal/service/project/resource_project.go (hunk 1 above)
Comment (ivan-savciuc) [reply internal/service/project/resource_project.go:72]: This is why, a user will be adding tag one be one 
```
                  tag {
		    key   = "test"
		    value = "val"
		  }
		  tag {
		    key   = "test"
		    value = "val2"
		  }
```

File: internal/service/project/resource_project.go (hunk 2 above)
Comment (ivan-savciuc) [reply internal/service/project/resource_project.go]: done

**Review by dmitrii-vasilev — APPROVED** [review]
LGTM

**Pull request merged**